<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMatch</class>
 <widget class="QDialog" name="FrmMatch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSGO - Demovoice - Match</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lblName">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtName">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblDemo">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Demo (.dem)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDemo">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDemoOpen">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Open</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblAudio">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Audio (.wav)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtAudio">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAudioOpen">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Open</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblTickrate">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tickrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboTickrate">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="currentIndex">
            <number>3</number>
           </property>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblDemoBegin">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Demo match begin (H/M/S)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtDemoBeginH">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtDemoBeginM">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtDemoBeginS">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Development in progress.</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblAudioBegin">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Audio match begin (H/M/S)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtAudioBeginH">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtAudioBeginM">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtAudioBeginS">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Development in progress.</string>
           </property>
           <property name="text">
            <string>Listen</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblStatus">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Please wait while the data are loaded/saved.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <layout class="QGridLayout" name="layoutLoading"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnSave</tabstop>
  <tabstop>txtName</tabstop>
  <tabstop>txtDemo</tabstop>
  <tabstop>btnDemoOpen</tabstop>
  <tabstop>txtAudio</tabstop>
  <tabstop>btnAudioOpen</tabstop>
  <tabstop>cboTickrate</tabstop>
  <tabstop>txtDemoBeginH</tabstop>
  <tabstop>txtDemoBeginM</tabstop>
  <tabstop>txtDemoBeginS</tabstop>
  <tabstop>btn</tabstop>
  <tabstop>txtAudioBeginH</tabstop>
  <tabstop>txtAudioBeginM</tabstop>
  <tabstop>txtAudioBeginS</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
